### DISCUSSION TITLE
Security Onion Rules

### DISCUSSION THREAD

**Discussion by Perry1000** [Security-Onion-Solutions/securityonion#10686] [Unsupported Versions] [answered]
Hi, I am new to security onion. I am currently evaluating it for a project. I have a management and sniffing interface. The sniffing interface consists of Kali and a metasploitable VM. After carrying out some attacks (Backdoor and SQL injection) on the metasploitable, I don't receive any alert on security onion. 
The alerts i have been receiving are nmap scans and kali hostname in DHCP request packet. 

In summary how can i configure security onions to alert when attacks are happening. I would appreciate your support.

**Comment by InfosecGoon** [comment, marked as answer]
By default, Security Onion installs with the IDS rules from the ET OPEN ruleset, meaning that you will get alerts for malicious network traffic that's identified in those rules. That's why you're getting the alerts for "kali hostname detected", because DHCP requests go out on the wire over an unencrypted channel and so the monitor port sees them, evaluates them against the ET OPEN rules in Suricata, and then generates the alert when there's a match.

Depending on what these attacks are that you're carrying out and whether they're occurring across a plaintext (port 80) connection, there may or may not be rules to catch them. Can you share some more information about how you're attacking this Metasploitable instance? Are you following a writeup or YouTube video from somewhere?

**Reply by Perry1000** [reply]
I am carrying some popular attacks which includes sql injection, Denial of Service, Malware injection, Reverse shell.
Yes i am following a YouTube video but they only showed the installation process.

Here is the setup:
There are 4 VMs
1. Metasploitable 
2. Kali
3. Security Onion
4. Ubuntu (Used to access seconion web interface

Security onion and ubuntu are on a Management network interface.
Metasploitable and kali are on a sniffing interface.

So when i carry out network scans, security onion is able to detect it. However when i carried out sql injection or backdoor attack on the metasploitable there was no alert.
It is also worth pointing out that when i typed "id" after gaining root access securityonion alerted that "check returned root"

Is there a way i can customise rules based on the signature i have identified from the packet.
